CONTRIBUTING.md

```diff
@@ -311,8 +311,8 @@ Place the `starboard-catalog.Dockerfile` in the top-level directory of your clon
 it to your Kubernetes cluster:
 
 ```
-$ docker build -f starboard-catalog.Dockerfile -t starboard-catalog:dev .
-$ kind load docker-image starboard-catalog:dev
+$ docker build -f starboard-catalog.Dockerfile -t docker.io/<your account>/starboard-catalog:dev .
+$ docker push docker.io/<your account>/starboard-catalog:dev
```

Comment:
In OLM v0.17.0 the catalog source pods use `imagePullPolicy: Always` instead of `IfNotPresent`. Hence, we must push the catalog image to pubic registry :(